ipld/blockstore/Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
 name = "ipld_blockstore"
-version = "0.1.0"
+description = "Filecoin storage interface"
```

Comment:
Its not really filecoin specific tho

Reply:
ah ya, was being speedy, I'll swap to Ipld storage